src/mode/modeHandler.ts

```diff
@@ -606,6 +606,7 @@ export class ModeHandler implements vscode.Disposable, IModeHandler {
 
     const recordedState = this.vimState.recordedState;
     recordedState.actionKeys.push(key);
+    VSCodeContext.set('vim.command', recordedState.commandString);
```

Comment:
Does this have a measurable impact on performance?

Reply:
I've been using it for 8 months now and haven't noticed any negative performance impact.

It's not being awaited here, so I'm sure that helps.